setup.py

```diff
@@ -38,7 +38,6 @@ def get_version(module='spyder_kernels'):
 REQUIREMENTS = [
     'cloudpickle',
     'ipykernel',
-    'ipykernel>=5.1.3; python_version>"2"',
```

Comment:
@ccordoba12 what should I leave here?

Reply:
The same, and please also revert the classifier deletions you did below.

Reply:
Ok